packages/react-components/src/components/styles/ReactionButton.styles.ts

```diff
@@ -29,12 +29,13 @@ export const playFrames = memoizeFunction(() =>
  * @private
  */
 export const emojiStyles = (backgroundImage?: string, animationPlayState?: string): React.CSSProperties => {
```

Comment:
nit: backgroundImage should be required here instead of being optional

Reply:
Updated according to suggestion. Thanks :)